rotkehlchen/tests/api/test_settings.py

```diff
@@ -582,3 +593,29 @@ def test_queried_addresses_per_protocol(rotkehlchen_api_server):
         'aave': [address1, address2],
         'makerdao_vaults': [address2],
     })
+
+
+@pytest.mark.parametrize('setting, default_price_oracles', [
+    ('current_price_oracles', DEFAULT_CURRENT_PRICE_ORACLES),
+    ('historical_price_oracles', DEFAULT_HISTORICAL_PRICE_ORACLES),
+])
+def test_set_price_oracles(rotkehlchen_api_server, setting, default_price_oracles):
```

Comment:
As I have said before for such easy tests it's better to just unit test. Bringing up the entire api server may be a bit of an overkill.

At least parametrize the eth accounts to be zero (they are not by default) so all ethereum intiialization stuff does not happen.